apps/condo/domains/ticket/utils/serverSchema/ticketBlankExport.js

```diff
@@ -0,0 +1,593 @@
+    }
+
+    const commentsWhere = {
+        ticket: { id_in: [ticket.id] },
+    }
+    if (!isEmpty(blankOptions.commentIds)) {
+        commentsWhere.id_in = blankOptions.commentIds
+    }
+
+    return await loadTicketCommentsForPdfExport({ where: commentsWhere })
+}
+
+const getTranslations = ({ locale, ticketBlankData }) => {
+    const getTranslate = getTranslateWithLocale({ locale })
+
+    return {
+        printDate: getTranslate('ticketBlankExport.label.printDate', { printDate: ticketBlankData.printDate }),
+        ticketWithNumberAndDated: getTranslate('ticketBlankExport.label.ticketWithNumberAndDated', {
+            number: ticketBlankData.number,
+            createdAt: ticketBlankData.createdAt,
+        }),
+        page: getTranslate('ticketBlankExport.label.page'),
+        ticket: getTranslate('ticketBlankExport.label.ticket'),
+        address: getTranslate('ticketBlankExport.heading.address'),
+        clientName: getTranslate('ticketBlankExport.heading.clientName'),
+        details: getTranslate('ticketBlankExport.heading.details'),
+        comments: getTranslate('ticketBlankExport.heading.comments'),
+        notes: getTranslate('ticketBlankExport.heading.notes'),
+        listCompletedWorks: getTranslate('ticketBlankExport.heading.listCompletedWorks'),
+        consumedMaterials: getTranslate('ticketBlankExport.heading.consumedMaterials'),
+        totalCostWork: getTranslate('ticketBlankExport.heading.totalCostWork'),
+        workCompleted: getTranslate('ticketBlankExport.message.workCompleted'),
+        executorFullName: getTranslate('ticketBlankExport.field.executorFullName'),
+        clientFullName: getTranslate('ticketBlankExport.field.clientFullName'),
+        executorSignature: getTranslate('ticketBlankExport.field.executorSignature'),
+        clientSignature: getTranslate('ticketBlankExport.field.clientSignature'),
+        completionDate: getTranslate('ticketBlankExport.field.completionDate'),
+    }
+}
+
+const generatePdf = (replaces) => {
+    const { i18n, blank } = replaces
+
+    const addComments = (comments) => {
+        if (!comments || comments.length < 1) return null
+
+        return {
+            marginBottom: 6,
+            stack: [
+                {
+                    style: 'subtitle',
+                    text: i18n.comments,
+                },
+                {
+                    stack: blank.comments.map((comment) => ({
+                        text: `${comment.createdAt} ${comment.content}`,
+                        fontSize: 8,
+                        marginBottom: 6,
+                    })),
+                },
+            ],
+        }
+    }
+
+    const addLines = (count, startPosition) => {
+        if (!count || !isNumber(count)) return null
+
+        return new Array(count).fill(1).map((_, index) => `<rect x="0" y="${startPosition + 14 + 15 * index}" width="400" height="1" fill="#707695"/>`)
+    }
+
+    const addBlockWithLines = (title, countLines) => {
```

Comment:
Grammatically, `countLines` seems like an action, but here it's a value

```suggestion
    const addBlockWithLines = (title, count) => {
```